src/relay/WalletSDKRelay.ts

```diff
@@ -1166,4 +1180,6 @@ export class WalletSDKRelay extends WalletSDKRelayAbstract {
         break;
     }
   }
+
+  protected onSessionConfigChanged(_nextSessionConfig: SessionConfig): void {}
```

Comment:
Need to add this to `WalletLinkSDKRelayAbstract` https://github.com/coinbase/coinbase-wallet-sdk/blob/master/src/relay/WalletSDKRelayAbstract.ts

Reply:
why?

Reply:
That's the class that the extension overrides.